### PR TITLE
civetweb: fix iOS build

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

Like usual, issue with missing BUNDLE DESTINATION for executables.

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
PR submitted upstream: https://github.com/civetweb/civetweb/pull/1015

**Review by prince-chrismc — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
> PR submitted upstream: [civetweb/civetweb#1015](https://github.com/civetweb/civetweb/pull/1015)

Amazing!

File: recipes/civetweb/all/patches/0001-fix-install-bundle.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -257,9 +257,7 @@ if (CIVETWEB_ENABLE_SERVER_EXECUTABLE)
+         install(
+             TARGETS civetweb-c-executable
+             EXPORT ${PROJECT_NAME}-targets
+-            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-            RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++            DESTINATION ${CMAKE_INSTALL_BINDIR}
```
Comment (SSE4) [inline recipes/civetweb/all/patches/0001-fix-install-bundle.patch:10]: so, `ARCHIVE`, `LIBRARY` and `RUNTIME` aren't needed?
I would expect something like:
```
            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
            RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
            BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}

```

**Review by SSE4 — APPROVED** [review]

File: recipes/civetweb/all/patches/0001-fix-install-bundle.patch (hunk 1 above)
Comment (SpaceIm) [reply recipes/civetweb/all/patches/0001-fix-install-bundle.patch:10]: Useless for an executable

**Pull request merged**